src/components/Trends/TrendsTable/TrendsTable.js

```diff
@@ -21,116 +24,134 @@ const columns = [
     Header: '#',
     accessor: 'index',
     width: 35,
-    headerClassName: styles.headerIndex
+    headerClassName: styles.headerIndex,
   },
   {
     Header: 'Word',
-    accessor: 'word'
+    accessor: 'word',
   },
   {
-    Header: 'Hyped score',
-    accessor: 'score'
+    Header: 'Hype score',
+    accessor: 'score',
   },
   {
     Header: 'Social volume',
-    accessor: 'volume'
-  }
+    accessor: 'volume',
+  },
 ]
 
-const NumberCircle = props => (
-  <div {...props} className={styles.insights__number} />
+const NumberCircle = ({ className, ...props }) => (
+  <div {...props} className={cx(className, styles.insights__number)} />
+)
+
+const Explanation = ({ text, children, ...props }) => (
```

Comment:
Is it possible common usage for many tooltips on our UI? @DmitriVanGuard

Reply:
Actually 2 or 3 places. So yeah, better to move it to the separate file 👍